examples/exec/Program.cs

```diff
@@ -0,0 +1,40 @@
+using k8s;
+using System;
+using System.Threading.Tasks;
+
+namespace exec
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            var pod = args[0];
+            var container = args.Length > 1 ? args[1] : null;
+
+            var k8SClientConfig = KubernetesClientConfiguration.BuildConfigFromConfigFile();
+            var client = new Kubernetes(k8SClientConfig);
+
+            var webSocket = await client.WebSocketNamespacedPodExecWithHttpMessagesAsync(pod, container: container, command: "/bin/bash").ConfigureAwait(false);
```

Comment:
Maybe the API cloud return `ExecClient` directly, make `websocket with channel` for intern use only